src/components/mgt-login/mgt-login.ts

```diff
@@ -228,33 +234,66 @@ export class MgtLogin extends MgtBaseComponent {
     if (!this.userDetails) {
       return;
     }
-    const avatarSize = 'large';
-
     return html`
       <div class="popup">
         <div class="popup-content">
           <div>
-            <mgt-person
-              .personDetails=${this.userDetails}
-              .personImage=${this._image}
-              .avatarSize=${avatarSize}
-              show-name
-              show-email
-            />
+            ${this.renderFlyoutPersonDetails(this.userDetails, this._image)}
           </div>
           <div class="popup-commands">
-            <ul>
-              <li>
-                <button class="popup-command" @click=${this.logout} aria-label="Sign Out">
-                  Sign Out
-                </button>
-              </li>
-            </ul>
+            ${this.renderFlyoutCommands()}
           </div>
         </div>
       </div>
     `;
   }
+
+  /**
+   * Render the flyout person details.
+   *
+   * @protected
+   * @returns
+   * @memberof MgtLogin
+   */
+  protected renderFlyoutPersonDetails(personDetails: IDynamicPerson, personImage: string) {
+    const template = this.renderTemplate('flyout-person-details', null);
```

Comment:
**flyout-person-details**
Needs the `personDetails` and `personImage` as stated in the documentation:
https://github.com/microsoftgraph/microsoft-graph-docs/pull/8086/files#diff-bd7d0ad3ffd332707b7c9931f3ec5828R89

Reply:
Wow, great catch - fixed